docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -259,27 +259,34 @@ You may notice that ``verdi process list`` now shows more than one entry:
 
 While you wait for the workflow to complete, let's start exploring its provenance.
 
-The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+Exploring the database
+----------------------
 
-Start the AiiDA REST API:
+In most cases, the full provenance graph obtained from ``verdi node graph generate`` will be rather complex to follow.
+To see this for yourself, you can try to generate the one for the workchains ran by the QuantumEspresso APP, or for the workchain script of the last section.
+It therefore becomes very useful to learn how to browse the provenance interactively instead.
+
+To do so, we need first to start the AiiDA REST API:
 
 .. code-block:: console
 
-    $ verdi restapi
+  $ verdi restapi
 
-Now, open a new terminal from the start page and run `ngrok`_, a tool that allows us to expose the REST API to a public URL:
+If you were working on your local machine, you would be automatically be able to access your exposed data via `http://127.0.0.1:5000/api/v4` (this would also work from inside the virtual machine).
```

Comment:
```suggestion
If you were working on your local machine, you would be automatically be able to access your exposed data via ``http://127.0.0.1:5000/api/v4`` (this would also work from inside a virtual machine).
```